packages/app/src/app/components/CreateNewSandbox/CreateSandbox/SearchBox/SearchBox.tsx

```diff
@@ -12,7 +12,7 @@ type SearchProps = {
 export const SearchBox = ({
   value,
   onChange,
-  placeholder = 'Search',
+  placeholder = 'Search Templates',
```

Comment:
```suggestion
  placeholder = 'Search templates',
```

Per design guidelines, we should default to sentence casing.